packages/kg-default-nodes/lib/utils/image-dimensions.js

```diff
@@ -1,4 +1,4 @@
-export const resizeImage = function (image, {width: desiredWidth, height: desiredHeight} = {}) {
+export const getImageDimensions = function (image, {width: desiredWidth, height: desiredHeight} = {}) {
```

Comment:
I wonder if this would be better named as `getResizedImageDimensions()` to better reflect what it's doing? `getImageDimensions` sounds like it's just reading dimensions from the image rather than calculating a new size.

Reply:
Thanks yes, that's even better.
I was originally thinking `getResizedDimensions`, but I think `getResizedImageDimensions` adds maximum clarity. :)